### ISSUE TITLE
Show Utility Buttons Vertically

### ISSUE THREAD

**Issue by qasimmehboob** [CEWendel/SWTableViewCell#390]
In tableview i have a cell with a big height. i want to show utility button vertically? How can i do that?

**Comment by pvn** [comment]
@qasimmehboob I have the similar question. Did you achieve that ? Can you please share your idea ?

**Comment by qasimmehboob** [comment]
@pvn : sorry i didn't find the solution. i was busy with some other stuff. if i solve this solution i'll let you know...

**Comment by KishanMakk** [comment]
@pvn,@qasimmehboob guys have u get the solution?

**Comment by pvn** [comment]
Yes, just hack. Setting up the background image on UIButton in manner to looks like Vertically aligned.
